src/main/groovy/edu/wpi/first/gradlerio/test/NativeExternalSimulationTask.groovy

```diff
@@ -106,4 +106,15 @@ class NativeExternalSimulationTask extends ExternalSimulationTask {
         outfile.parentFile.mkdirs()
         outfile.text = json
     }
+
+    @Internal
+    boolean includeTestBin = true
```

Comment:
Don't mark this as @Internal Thats only for non user accessable properties.

Also just make this be `boolean includeTestBinaries = true` and remove the function. Declarations like this are properties in groovy, not fields.